source/windows/file.c

```diff
@@ -31,3 +37,106 @@ FILE *aws_fopen(const char *file_path, const char *mode) {
     }
     return file;
 }
+
+char aws_get_platform_directory_separator(void) {
+    return '\\';
+}
+
+AWS_STATIC_STRING_FROM_LITERAL(s_userprofile_env_var, "USERPROFILE");
+AWS_STATIC_STRING_FROM_LITERAL(s_homedrive_env_var, "HOMEDRIVE");
+AWS_STATIC_STRING_FROM_LITERAL(s_homepath_env_var, "HOMEPATH");
+
+AWS_STATIC_STRING_FROM_LITERAL(s_home_env_var, "HOME");
+
+struct aws_string *aws_get_home_directory(struct aws_allocator *allocator) {
+
+    /*
+     * 1. Check HOME
```

Comment:
does this cover all unix platforms since the other two look only windows related?

Reply:
yeah, it's POSIX